.github/workflows/test-e2e-windows.yml

```diff
@@ -182,6 +182,15 @@ jobs:
       - name: Send Results to GH Summary
         uses: ./.github/actions/gen-report-dir
 
+      - name: Run Extensions Test
```

Comment:
Run first to get early feedback before E2E `test,` as this step is quick to complete. But don't prevent E2E tests from running when it fails, only fail at the end.